plugins/hosts/windows/cap/rdp.rb

```diff
@@ -21,8 +21,10 @@ def self.rdp_client(env, rdp_info)
           if rdp_info[:extra_args]
             args = rdp_info[:extra_args] + args
           end
-          # Launch it
-          Vagrant::Util::Subprocess.execute("mstsc", *args)
+          # Spawn it and detach from process. This will allow
+          # Vagrant to exit independent from the rdp session
+          pid = spawn("mstsc", *args)
```

Comment:
[childprocess 4.0.0](https://github.com/enkessler/childprocess/releases/tag/v4.0.0) supports detaching from a process in Windows. Might be nice to update that gem and make Vagrant detach from the process that way?

Reply:
I think that sounds like the best path, and once the dependency constraint is updated, adding in functionality to the subprocess util to handle detaching from the created process would be much nicer.